servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/DefaultGrpcClientBuilder.java

```diff
@@ -84,6 +84,8 @@ public GrpcClientBuilder<U, R> defaultTimeout(@Nullable final Duration defaultTi
 
     @Override
     public <Client extends GrpcClient<?>> Client build(GrpcClientFactory<Client, ?> clientFactory) {
+        // TODO: how can we get the response classifier down to the LoadBalancer since we don't even know
+        //  what load balancer we have?
```

Comment:
We won't be able to do anything in grpc package, it doesn't have required details. The way we discussed an approach with you offline was to use `HttpLifecycleObserver` inside `http-netty` (the same one you created there) and enhance it to look for `grpc-status` header or trailer using its callbacks.

Reply:
Indeed, that is what we had discussed but starts encoding grpc specific detail into the http package which seems like a mistake in it's own right and also makes it difficult to share the conversions.

Reply:
So there is a way to add it, but it's pretty hairy. It requires adding a decorator `SingleAddressLoadBalancerBuilder` that can intercept calls to `loadBalancerFactory(..)` and wrap the result with a special `HttpLoadBalancerFactory` with a special `toLoadBalancedConnection(..)` method that will install the tracker. It works as intended but it generally feels like the `toLoadBalancedConnection(..)` method is being abused despite this appearing to be what it's intended for.